packages/expo-modules-core/android/src/main/java/expo/modules/kotlin/types/TypeConverterProvider.kt

```diff
@@ -68,9 +69,15 @@ object TypeConverterProviderImpl : TypeConverterProvider {
       return EnumTypeConverter(kClass as KClass<Enum<*>>, type.isMarkedNullable)
     }
 
-    // TODO(@lukmccall): cache record type converters
+    val maybeConverter = cachedRecordConverters[kClass]
+    if (maybeConverter != null) {
+      return maybeConverter
+    }
```

Comment:
`maybe` sounds kinda weird here 🙈 Why not `cached` instead?

```suggestion
    val cachedConverter = cachedRecordConverters[kClass]
    if (cachedConverter != null) {
      return cachedConverter
    }
```